CONST_vars.yaml

```diff
@@ -231,7 +235,7 @@ vars:
     # Checker configuration
     checker:
         fulltextsearch: text to search
-        lang_files: [ngeo, cgxp, cgxp-api]
+        lang_files: [cgxp, cgxp-api]
```

Comment:
Yes because it we add it in the version 1.6 we should modify all the project ...
I think that you just should add 
`lang_files: [ngeo]`
In the lux project!